forms-flow-api/src/formsflow_api/resources/cmis_connector.py

```diff
@@ -39,15 +39,22 @@ def post():
             uploads = repo.getObjectByPath("/uploads")
 
         if 'upload' not in request.files:
```

Comment:
```suggestion
        if not hasattr(request.files, "upload"):
```

https://docs.python.org/3/library/functions.html#hasattr

Reply:
assuming request.files is an object